Sources/SwiftSyntaxBuilder/ResultBuilderExtensions.swift

```diff
@@ -50,20 +50,14 @@ extension MemberBlockItemListBuilder {
   }
 }
 
-// MARK: Initializing collections from protocols
-// These initializers allow the creation of syntax collections that have a base
-// node as their element from the corresponding protocol type.
-// These are used by the result builders.
-// Since we only have two of these, it doesn’t make sense to generate them.
-
-extension ExprListSyntax {
-  init(_ elements: [ExprSyntaxProtocol]) {
-    self = ExprListSyntax(elements.map { ExprSyntax(fromProtocol: $0) } as [ExprSyntax])
```

Comment:
I think we still need this so that you can write eg. 

```swift
ExprListSyntax([DeclReferenceExprSyntax(…), ForceUnwrapExprSyntax(…)])`
```

Or any heterogeneous array of expressions, which is what we do in the tests you changed, I think.
Otherwise this is an API-breaking change.

Same for the initializer on `UnexpectedNodesSyntax` below.

Reply:
Yeah, no problem. I will put it back 🙂